NAMESPACE

```diff
@@ -52,6 +71,8 @@ importFrom(checkmate,check_numeric)
 importFrom(cli,cli_abort)
 importFrom(cli,cli_alert_danger)
 importFrom(cli,cli_alert_info)
+importFrom(cli,cli_inform)
```

Comment:
Note I probably need to edit some `alert_info` --> `inform` in other places but can be a separate PR (sorry didn't realize there were other options for messaging). E.g. in the `prop_delay` rounding for instance